Makefile

```diff
@@ -162,14 +149,68 @@ full-reset: docker-compose-rm clean all
 
 .PHONY: docker
 docker:
-	docker build -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) .
+	docker buildx build \
+		--load \
+		-t "$(DOCKER_TAG)" \
+		-t "$(DOCKER_TAG):parent" \
+		--target=$(DOCKER_TARGET) \
+		--build-arg=GIT_VERSION=$(GIT_VERSION) \
+		--build-arg=BUILD_TARGET=$(BUILD_TARGET) \
+		--build-arg=FROM_LOCAL_PARENT=$(FROM_LOCAL_PARENT) \
+		$(ARGS) \
+		.
 
 .PHONY: docker-local
-docker-local:
-	tar ch ./bin Dockerfile.local | docker build -f Dockerfile.local -t "$(DOCKER_TAG)" --build-arg=GIT_VERSION=$(GIT_VERSION) --build-arg=BUILD_TARGET=$(BUILD_TARGET) -
-
+docker-local: downloader livepeer-log scripts 
+	tar ch ./bin Dockerfile.local ./config \
```

Comment:
Aren't you removing `Dockerfile.local` in this PR?

Reply:
Good point. 9673737